pages/support/endowments/index.vue

```diff
@@ -0,0 +1,82 @@
+<template lang="html">
+    <main
+        class="page page-endowment-listing"
+    >
+        <nav-breadcrumb
+                to="/support"
+                title="Endowments"
+                parent-title="Support Us"
+            />
+
+        <masthead-secondary
+            :title="page.title"
+            :text="page.summary"
+        />
+
+        <!-- <search-generic
+            search-type="about"
+            :filters="searchFilters"
+            class="generic-search"
+            @search-ready="getSearchData"
+        /> -->
+
+        <section-wrapper theme="divider">
+            <divider-way-finder
+                color="about"
+            />
+        </section-wrapper>
+
+        <section-wrapper>
```

Comment:
The links on the titles are going to `/undefined`

Reply:
updated query